### DISCUSSION TITLE
Rasa chatbot not showing Payload Buttons

### DISCUSSION THREAD

**Discussion by manojdhimann** [chatwoot/chatwoot#6586] [Q&A]
![image](https://user-images.githubusercontent.com/32815842/222082418-86716836-6ac4-485b-aa81-2b9178c4bb27.png)

Actual Value: 
![image](https://user-images.githubusercontent.com/32815842/222082644-ff760f72-4f66-4303-8bb6-c8e4b3148992.png)

**Comment by manojdhimann** [comment]
**php service logs**

[Wed Mar  1 08:32:10 2023] message_type: incoming
[Wed Mar  1 08:32:10 2023] message: hi
[Wed Mar  1 08:32:10 2023] conversation: 61
[Wed Mar  1 08:32:10 2023] contact: 63
[Wed Mar  1 08:32:10 2023] account: 1
[Wed Mar  1 08:32:10 2023] sending message to bot: hi
[Wed Mar  1 08:32:10 2023] bot replied: what would you prefer:
[Wed Mar  1 08:32:10 2023] chatwoot response: {"id":134,"content":"what would you prefer:","inbox_id":1,"conversation_id":61,"message_type":1,"content_type":"text","status":"sent","content_attributes":{},"created_at":1677659530,"private":false,"source_id":null,"sender":{"id":1,"name":"Widget Talk","avatar_url":"","type":"agent_bot"}}
[Wed Mar  1 08:32:10 2023] 172.19.0.4:49140 [200]: POST /
[Wed Mar  1 08:32:10 2023] 172.19.0.4:49140 Closing
[Wed Mar  1 08:32:10 2023] 172.19.0.4:49146 Accepted

**Comment by manojdhimann** [comment]
@sojan-official Is that possible or not?

**Comment by manojdhimann** [comment]
Did you guys check?

**Comment by pranavrajs** [comment]
@manojdhimann Have you tried the interactive messages option in Chatwoot? Based on the Rasa bot output you can customize your payload to show the buttons. Please refer. https://www.chatwoot.com/docs/product/others/interactive-messages

**Reply by pranavrajs** [reply]
I moved this to Q&A discussions from issues.

**Comment by manojdhimann** [comment]
Do we have any sample Rasa code with mentioned payload templates?

https://www.chatwoot.com/docs/product/others/interactive-messages

**Comment by manojdhimann** [comment]
Hey @pranavrajs 

I'm using user profile base access token. 
Is that right? I'm getting this error.  {"errors":["You need to sign in or sign up before continuing."]}⏎

curl -X POST \
  https://app.*****.com/api/v1/accounts/1/conversations/2/messages \
  -H 'Content-Type: application/json' \
  -d '{
    "api_access_token": "**********",
    "payload": {
      "content": "string",
      "message_type": "outgoing",
      "private": true,
      "content_type": "cards",
      "content_attributes": {}
    }
}'
